denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc

```diff
@@ -0,0 +1,45 @@
+armor_degradation:
```

Comment:
This looks like a copy-paste from the last script you contributed per your last pull request
```suggestion
slime_mob_handler:
```